tests/python/common/test_convert.py

```diff
@@ -0,0 +1,179 @@
+from collections import defaultdict as ddict
+
+import dgl
+import networkx as nx
+
+import backend as F
+
```

Comment:
Define a single `test_to_networkx` so that we have a one-on-one correspondence between the unit test function and the user facing API.

Reply:
Sounds good. Should this contain only the test code from the new `test_nx_convert_heterogeneous` (maybe with a TODO mentioning the other existing test, which I'd move back to `test_basics` until the `DGLError` issue is solved)?